### PR TITLE
Fix no such configure file when HEAD build

### PR DESCRIPTION
**Pull request by zchee**
Signed-off-by: Koichi Shiraishi <zcheeee@gmail.com>

### CODE REVIEW

File: Library/Formula/pcre.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Pcre < Formula
   def install
     ENV.universal_binary if build.universal?
 
+    system "./autogen.sh" if build.head?
```
Comment (DomT4) [inline Library/Formula/pcre.rb]: Presumably this needs the autotools, in which case you'll need to add a `head` block with the autotools as build dependencies.

**Comment by zchee** [comment]
@DomT4 Yes, sir!!
Will fix!

**Comment by zchee** [comment]
Fixed!

But, required `--env=std` flag.
If do not set this flag, return `./autogen.sh: line 33: aclocal: command not found` errors.

**Comment by DomT4** [comment]
Sorry if I confused you here :). You want to do something like this:

```ruby
head do
  url "svn://vcs.exim.org/pcre/code/trunk"

  depends_on "automake" => :build
  depends_on "autoconf" => :build
  depends_on "libtool" => :build
end
```

```ruby
def install
ENV.universal_binary if build.universal?

system "./autogen.sh" if build.head?
```

You've already got the last bit done.

**Comment by zchee** [comment]
@DomT4 Fixed.
Will that be OK?

**Comment by DomT4** [comment]
@zchee Looks perfect, Thanks!

**Comment by zchee** [comment]
Thanks advise :)

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message is `pcre: fix HEAD build.` and please don't sign-off your own commits.

**Pull request closed without merging**